src/HttpKernel/InstallationKernel.php

```diff
@@ -67,7 +67,6 @@ public function canBootRealSystem()
     {
         return file_exists($this->getRootDir().'/config/parameters.yml')
             && file_exists($this->getRootDir().'/../system/config/localconfig.php')
```

Comment:
Oh, and if `Config::has('licenseAccepted')` is not set, the real system will throw an `IncompleteInstallationException`.